### PR TITLE
Match BattleIntroOpponent1SendsOutMonAnimation

### PR DESCRIPTION
**Pull request by PokeCodec**
Thanks everyone! With this, all the naked functions are done! We only have fakematches left.

However, the includes are going to have to be cleaned up, but otherwise, this is a monumental milestone!

### CODE REVIEW

File: berry_fix/Makefile [hunk 1]
```diff
@@ -42,7 +42,7 @@ CPPFLAGS := -I ../tools/agbcc/include -I ../tools/agbcc -iquote include -nostdin
 ROM := berry_fix.gba
 OBJ_DIR := build
 CC1             := ../tools/agbcc/bin/agbcc$(EXE)
-override CFLAGS += -mthumb-interwork -Wimplicit -Wparentheses -Werror -O2 -fhex-asm
+override CFLAGS += -mthumb-interwork -Wimplicit -Wparentheses -Werror -O2 -fhex-asm -nostdlib
```
Comment (PikalaxALT) [inline berry_fix/Makefile:45]: Revert

File: berry_fix/payload/Makefile [hunk 2]
```diff
@@ -38,7 +38,7 @@ CPPFLAGS := -I ../../tools/agbcc/include -I ../../tools/agbcc -iquote include -n
 ROM := payload.gba
 OBJ_DIR := build
 CC1             := ../../tools/agbcc/bin/agbcc$(EXE)
-override CC1FLAGS += -mthumb-interwork -Wimplicit -Wparentheses -Werror -O2 -fhex-asm
+override CC1FLAGS += -mthumb-interwork -Wimplicit -Wparentheses -Werror -O2 -fhex-asm -nostdlib
```
Comment (PikalaxALT) [inline berry_fix/payload/Makefile:41]: Revert

File: berry_fix/Makefile [hunk 3]
```diff
@@ -56,7 +56,7 @@ C_BUILDDIR = $(OBJ_DIR)/$(C_SUBDIR)
 ASM_BUILDDIR = $(OBJ_DIR)/$(ASM_SUBDIR)
 DATA_ASM_BUILDDIR = $(OBJ_DIR)/$(DATA_ASM_SUBDIR)
 
-ASFLAGS := -mcpu=arm7tdmi
+ASFLAGS := -mcpu=arm7tdmi -mthumb-interwork
```
Comment (PikalaxALT) [inline berry_fix/Makefile:59]: Revert

File: berry_fix/payload/Makefile [hunk 4]
```diff
@@ -52,7 +52,7 @@ C_BUILDDIR = $(OBJ_DIR)/$(C_SUBDIR)
 ASM_BUILDDIR = $(OBJ_DIR)/$(ASM_SUBDIR)
 DATA_ASM_BUILDDIR = $(OBJ_DIR)/$(DATA_ASM_SUBDIR)
 
-ASFLAGS := -mcpu=arm7tdmi
+ASFLAGS := -mcpu=arm7tdmi -mthumb-interwork
```
Comment (PikalaxALT) [inline berry_fix/payload/Makefile:55]: Revert

File: libagbsyscall/Makefile [hunk 5]
```diff
@@ -31,7 +31,7 @@ else
 EXE :=
 endif
 
-ASFLAGS := -mcpu=arm7tdmi
+ASFLAGS := -mcpu=arm7tdmi -mthumb-interwork
```
Comment (PikalaxALT) [inline libagbsyscall/Makefile:34]: Revert

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

**Comment by PokeCodec** [comment]
Done!

**Review by PikalaxALT — APPROVED** [review]

**Comment by PikalaxALT** [comment]
Ladies and gentlemen, the Wicked Witch of the HandleAction_UseItem is dead

**Pull request merged**